vertical-pod-autoscaler/deploy/vpa-crd.yaml

```diff
@@ -26,31 +26,11 @@ spec:
               properties:
                 updateMode:
                   type: string
-                  enum:
-                    - "Off"
-                    - "Initial"
-                    - "Recreate"
-                    - "Auto"
             resourcePolicy:
               properties:
                 containerPolicies:
                   type: array
                   items:
```

Comment:
After removing lines below items will be empty. Should be removed too?

Reply:
Done